WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/LoginFlow.java

```diff
@@ -39,7 +43,66 @@ public void login() {
         // Login Confirmation Screen – Click "Continue"
         // See LoginEpilogueFragment
         clickOn(R.id.primary_button);
+    }
+
+    private void confirmLogin() {
+        ViewInteraction continueButton = onView(withText(getCurrentActivity().getString(R.string.login_continue)));
+
+        // TODO: Needs to wait a little for button to be clickable and for load after, should try and find a better way
+        // On poor WiFi this needed to be increased to pass
```

Comment:
If this failed on poor wifi, it'll fail again on _even poorer_ wifi. My suggestion would be to use `waitForElementToBeDisplayed` to have the app pause until the operation is complete. If there's a progress spinner or something, you can use `waitForElementToNotBeDisplayed`

Reply:
> If there's a progress spinner or something, you can use waitForElementToNotBeDisplayed

Got it, yeah I'll update this!